[2025-01-30T10:25:44.205+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-30T10:25:44.333+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fhv_to_gcp_dag.upload_to_cloud scheduled__2019-11-02T00:00:00+00:00 [queued]>
[2025-01-30T10:25:44.368+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fhv_to_gcp_dag.upload_to_cloud scheduled__2019-11-02T00:00:00+00:00 [queued]>
[2025-01-30T10:25:44.368+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-30T10:25:46.020+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): upload_to_cloud> on 2019-11-02 00:00:00+00:00
[2025-01-30T10:25:46.060+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=302) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-30T10:25:46.066+0000] {standard_task_runner.py:72} INFO - Started process 322 to run task
[2025-01-30T10:25:46.068+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fhv_to_gcp_dag', 'upload_to_cloud', 'scheduled__2019-11-02T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/homework_fhv_dag.py', '--cfg-path', '/tmp/tmpln_spgg3']
[2025-01-30T10:25:46.118+0000] {standard_task_runner.py:105} INFO - Job 75: Subtask upload_to_cloud
[2025-01-30T10:25:46.663+0000] {task_command.py:467} INFO - Running <TaskInstance: fhv_to_gcp_dag.upload_to_cloud scheduled__2019-11-02T00:00:00+00:00 [running]> on host 9573d401388f
[2025-01-30T10:25:47.133+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fhv_to_gcp_dag' AIRFLOW_CTX_TASK_ID='upload_to_cloud' AIRFLOW_CTX_EXECUTION_DATE='2019-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-11-02T00:00:00+00:00'
[2025-01-30T10:25:47.135+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-30T10:25:49.533+0000] {logging_mixin.py:190} INFO - The file /opt/***/data/fhv_2019-11_data.parquet has been uploaded to nrabbi-33-dtc-de-data-bucket/<Blob: nrabbi-33-dtc-de-data-bucket, /data/fhv_2019-11_data.parquet, 1738232749553911>
[2025-01-30T10:25:49.533+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-30T10:25:49.544+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-30T10:25:49.545+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fhv_to_gcp_dag, task_id=upload_to_cloud, run_id=scheduled__2019-11-02T00:00:00+00:00, execution_date=20191102T000000, start_date=20250130T102544, end_date=20250130T102549
[2025-01-30T10:25:50.034+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-30T10:25:50.071+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-30T10:25:50.090+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
